packages/react/src/primitives/Table/Table.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import * as React from 'react';
+
+import { Flex, Text, View } from '..';
+import { ComponentClassNames } from '../shared/constants';
+import { TableProps } from '../types/table';
+
+export const Table: React.FC<TableProps> = ({
+  caption,
+  children,
+  className,
+  highlightOnHover = false,
+  label,
+  size = '',
+  summary,
+  variation = '',
+  ...rest
+}) => (
+  <Flex>
```

Comment:
What's the purpose of the outer `Flex`? It seems unnecessary

Reply:
Good call out, I removed it.